Sources/NIO/FileRegion.swift

```diff
@@ -22,11 +22,11 @@
 /// One important note, depending your `ChannelPipeline` setup it may not be possible to use a `FileRegion` as a `ChannelHandler` may
 /// need access to the bytes (in a `ByteBuffer`) to transform these.
 ///
-/// - note: It is important to manually manage the lifetime of the `FileHandle` used to create a `FileRegion`.
+/// - note: It is important to manually manage the lifetime of the `NIOFileHandle` used to create a `FileRegion`.
 public struct FileRegion {
```

Comment:
Should this be `NIOFileRegion` to match?

Reply:
good question. @normanmaurer / @Lukasa how do we feel about

- (`NIO`)`FileRegion`
- (`NIO`)`FileDescriptor`
- etc

Reply:
Fine by me, it's in line with our other type names.

Reply:
@normanmaurer ?

Reply:
Sure why not... that said I am not sure where to draw the line here. Do we now prefix everything ?

Reply:
@normanmaurer so I'm definitely against namespacing everything. I made my PR only namespace `FileHandle` because `NIO.FileHandle` does clash with `Foundation.FileHandle`.

Reply:
I see... that said I think there is really no guarantee that we will not have new clashes in the future as they may add new stuff over time.

Reply:
@normanmaurer correct, but their pace in adding API is a lot slower than ours :). And we're in the source compatibility suite so they'd notice as things like Vapor/Kitura/Smoke/... would fail to compile.

Reply:
To be clear, we do propose to namespace all new public types though, right?

Reply:
@Lukasa only new public types without releasing a new _major_ version. So as soon as NIO 2.0.0 is released, we will prefix everything until at least NIO 3.0.0 is released.